server/src/main/java/org/opensearch/index/remote/RemoteStoreUtils.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index.remote;
+
+import java.util.Arrays;
+
+/**
+ * Utils for remote store
+ */
+public class RemoteStoreUtils {
```

Comment:
nit: Call it PrefixConverter

Reply:
this is already merged . I can take it up once remote translog and segment changes are merged .